db/schema.rb

```diff
@@ -13,7 +13,7 @@
 
 ActiveRecord::Schema.define(version: 20141215103539) do
 
-  create_table "comments", force: true do |t|
+  create_table "comments", force: :cascade do |t|
```

Comment:
where did this changes come from? Will this somehow apply to the production database, or do we have to recreate it?